tests/Nox.Lib.Tests/Factories/Types/NoxTypeCultureCodeFactoryTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using FluentAssertions;
+using Nox.Factories.Types;
+using Nox.Lib.Tests.FixtureConfig;
+using Nox.Solution;
+using Nox.Types;
+
+namespace Nox.Lib.Tests.Factories.Types
+{
+    public class NoxTypeCultureCodeFactoryTests
+    {
+        [Theory, AutoMoqData]
+        public void CreateNoxType_FromDto_IsValid(NoxSolution noxSolution, EntityDefinitionFixture fixture)
+        {
+            // Arrange
+            var sut = new NoxTypeCultureCodeFactory(noxSolution);
+            var cultureCode = "en-US";
+
+            // Act
+            var entity = sut.CreateNoxType(fixture.EntityDefinition, fixture.PropertyName, cultureCode);
+
+            // Assert
+            entity.Should().NotBeNull();
+            entity!.Value.Should().Be(cultureCode);
+        }
+
+        [Theory, AutoMoqData]
+        public void CreateNoxType_FromDto_WhenValueIsNull_IsValid(NoxSolution noxSolution, EntityDefinitionFixture fixture)
+        {
+            // Arrange
+            var sut = new NoxTypeCultureCodeFactory(noxSolution);
+            string? cultureCode = null;
+
+            // Act
+            var entity = sut.CreateNoxType(fixture.EntityDefinition, fixture.PropertyName, cultureCode);
+
+            // Assert
+            entity.Should().BeNull();
+        }
+
+        [Theory, AutoMoqData]
+        public void CreateNoxType_FromDto_WhenValueIsInvalid_IsValid(NoxSolution noxSolution, EntityDefinitionFixture fixture)
```

Comment:
You forgot to change the `IsValid` from the method name.

Reply:
Fixed. Thanks.